packages/webapp/src/containers/Filter/Transactions/index.jsx

```diff
@@ -61,16 +63,18 @@ const TransactionFilterContent = ({
         value: type.revenue_type_id,
         // This sets the initial state of the filter pill
         default: transactionsFilter[REVENUE_TYPE]?.[type.revenue_type_id]?.active ?? true,
-        label: type.farm_id
-          ? type.revenue_name
-          : t(`revenue:${type.revenue_translation_key}.REVENUE_NAME`),
+        label:
+          (type.farm_id
+            ? type.revenue_name
+            : t(`revenue:${type.revenue_translation_key}.REVENUE_NAME`)) +
+          (type.retired ? ` ${t('REVENUE.EDIT_REVENUE.RETIRED')}` : ''),
       })),
     },
   ];
 
   return (
     <FilterGroup
-      filters={filters}
+      filters={[sortFilterOptions(filters[0]), sortFilterOptions(filters[1])]}
```

Comment:
NIT: 

```suggestion
      filters={filters.map(filter => sortFilterOptions(filter))}
```

so that if we ever add new filters we don't have to go back and edit this line